CRM/Admin/Form/SettingTrait.php

```diff
@@ -290,7 +298,7 @@ protected function getQuickFormType($spec) {
       'advmultiselect' => 'Element',
     ];
     $mapping += array_fill_keys(CRM_Core_Form::$html5Types, '');
-    return $mapping[$htmlType];
+    return $mapping[$htmlType] ?? '';
```

Comment:
The upstream code checks with isset and skips all the adds.

Reply:
I feel like I deliberately didn't have enotice checking here because the expectation was that it should be set  & it should otherwise be noisy

Reply:
Ah - all the others are added to the mapping in the row above - that makes sense - we should explicitly add datepicker to $mapping rather than suppress notices here IMHO